library/deletedrug.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ *  @package   OpenEMR
+ *  @link      http://www.open-emr.org
+ *  @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c )2020. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ *
+ */
+
+require_once "../interface/globals.php";
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Acl\AclMain;
+
+//ensure user has proper access
+if (!AclMain::aclCheckCore('admin', 'acl', 'super', 'patients')) {
+    echo xlt('ACL Administration Not Authorized');
+    exit;
+}
+
+$id = filter_input(INPUT_POST, 'drugId', FILTER_VALIDATE_INT);
+$id = trim($id);
+if (isset($id)) {
+    if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"])) {
+        CsrfUtils::csrfNotVerified();
+        throw new Exception("Form validation failed");
```

Comment:
Is this exception doing anything? (note the script has done a die in the line above)

Reply:
I did not question the request to add it. According to what you told me in the beginning. The script should die if not verified. I thought I was being told something new. I will remove it.